<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_server_pointing</class>
 <widget class="QMainWindow" name="New_server_pointing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新增服务器指向</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>601</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>84</width>
       <height>171</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border : none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>协议类型</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>连接名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>服务器IP</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(50, 50, 51);</string>
        </property>
        <property name="text">
         <string>端口</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>284</width>
       <height>171</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>95</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
	border : none;
}
QLineEdit
{
	border : none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
	
	color: rgb(50, 50, 51);
	border : none;
}
QComboBox::down-arrow{
	image: url(:/png/shuncom_ico/png_ico/下拉4.png);
	hight : 16px;
	width : 16px;
	
}
QComboBox::drop-down{
	border-style: none;
}</string>
        </property>
        <item>
         <property name="text">
          <string>sziot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Touchuan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MQTT</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:10pt;
    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color:rgb(255,255,255);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>76</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>76</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:Microsoft Yahei;
    /*字体大小为20点*/
    font-size:10pt;
    /*字体颜色*/ 
	color: rgb(50, 50, 51);
 	/*背景颜色*/  
    background-color: rgb(21, 133, 212);
    /*边框圆角半径为8像素*/ 
    border-radius:2px;
	width:65px;
}
 
/*按钮停留态*/
QPushButton:hover
{
   /*背景颜色*/  
    background-color: rgb(85, 170, 255);
}
 
/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
 	background-color:rgb(170 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../shuncom_qrc.qrc"/>
 </resources>
 <connections/>
</ui>
